src/symbols.jl

```diff
@@ -596,7 +601,11 @@ function collect_extended_methods(mod::ModuleStore, extendeds, mname)
     end
 end
 
-getallns() = let allns = Base.IdSet{Symbol}(); oneverything((m, s, x, state) -> push!(allns, s)); allns end
+getallns() =
+    let allns = Base.IdSet{Symbol}()
+        oneverything((m, s, x, state) -> push!(allns, s))
+        allns
+    end
```

Comment:
These line breaks should not be added.

Reply:
just to clarify you mean the line break where there semicolon was?

Reply:
Yes, and I guess also the line break after the first `=` shouldn't be added. I just should stay one line as in the original.